inference-engine/tests/functional/plugin/cpu/single_layer_tests/convolution.cpp

```diff
@@ -190,17 +240,95 @@ const std::vector<SizeVector> strides2d = { {1, 1}, {2, 2} };
 const std::vector<std::vector<ptrdiff_t>> padBegins2d = { {0, 0}, {1, 1} };
 const std::vector<std::vector<ptrdiff_t>> padEnds2d = { {0, 0} };
 const std::vector<SizeVector> dilations2d = { {1, 1}, {2, 2} };
-const std::vector<SizeVector> inputShapes2d = { {1, 64, 7, 7}, {1, 67, 7, 7} };
-const std::vector<SizeVector> inputShapesPlain2Blocked2d = { {1, 1, 7, 7}, {1, 2, 7, 7},  {1, 3, 7, 7} };
+std::vector<InputShape> inputShapes2d = {
```

Comment:
As it was discussed offline we need to reduce number of testcases because dynamism support dramatically increases SLT  execution time. I would propose to exclude some parameters from pre-commit stage (for example non unit dilation).

Reply:
Done. Dilated convolution tests have been extracted from the precommit test run.